src/rt/sections_osx.d

```diff
@@ -23,6 +23,13 @@ import core.sys.osx.mach.getsect;
 import rt.deh, rt.minfo;
 import rt.util.container.array;
 
+version (D_LP64)
+    import rt.sections_osx64;
+else
+    import rt.sections_osx32;
+
+mixin SectionsOSX;
```

Comment:
So you mean a fourth file? There's already `section_osx.d`, `section_osx32.d` and `section_osx64.d`.

I'll wait for some additional input before making any changes.